setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name="ansible-runner",
-    version="1.3.1",
+    version="1.5.0",  # .. so I can tell if SLOWYO installed correct version
```

Comment:
This will almost certainly need to be removed before final acceptance.

Reply:
Reverted to original version